src/components/SuccesPage.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import Logo from "./Logo.png";
+import Message from "./MessageLogo.png";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
+import { Grid } from "@material-ui/core";
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      backgroundColor: "#FFFFFF",
+      maxWidth: "414px",
+      height: "836px",
+      marginTop: "30px",
+      paddingLeft: "25px",
+      "& .MuiGrid-spacing-xs-6 > .MuiGrid-item": {
+        paddingTop: "3px",
+      },
+    },
+
+    logo: {
+      marginTop: "34px",
+      marginBottom: "30px",
+      marginLeft: "10px",
+      width: "96px",
+      height: "30px",
+    },
+    message: {
+      marginTop: "44px",
+      marginBottom: "0px",
+      marginLeft: "10px",
+      width: "71px",
+      height: "53.99px",
+    },
+  })
+);
+function SuccesPage(props: any) {
```

Comment:
add empty line here as the above section is css section and below section is component